docs/cli.md

```diff
@@ -774,17 +774,9 @@ This command exports the lock file to other formats.
 poetry export -f requirements.txt --output requirements.txt
 ```
 
-{{% warning %}}
-This command is provided by the [Export Poetry Plugin](https://github.com/python-poetry/poetry-plugin-export).
-In a future version of Poetry this plugin will not be installed by default anymore.
-In order to avoid a breaking change and make your automation forward-compatible,
-please install poetry-plugin-export explicitly.
-See [Using plugins]({{< relref "plugins#using-plugins" >}}) for details on how to install a plugin.
-{{% /warning %}}
-
 {{% note %}}
-This command is also available as a pre-commit hook.
-See [pre-commit hooks]({{< relref "pre-commit-hooks#poetry-export" >}}) for more information.
+This command is provided by the [Export Poetry Plugin](https://github.com/python-poetry/poetry-plugin-export)
+and is also available as a pre-commit hook. See [pre-commit hooks]({{< relref "pre-commit-hooks#poetry-export" >}}) for more information.
```

Comment:
Maybe, we should make this a warning and put it at top of the `export` section.

Reply:
Suggestion: Since this will be a big change (removing the export plugin by default) perhaps it's worth adding some explicit docs on how to add this back in a project-specific plugin dependency (e.g. #9547)? I suspect this will be a very common need for many users to figure out.

Reply:
Good idea. Here is the relevant section in the docs: https://github.com/python-poetry/poetry/blob/b61e6bd974b7c584b12fa43fe983087cb507c94d/docs/plugins.md?plain=1#L258-L270

We can add a link or even a code snippet with `poetry-plugin-export` to be more explicit.

Reply:
@johnthagen I added some additional information. Please take a look.

Reply:
@radoering The new docs in c3d14384ffc4e3cf00c03639ed330ed0952d14af look great.